cypress/integration/register_spec.js

```diff
@@ -5,8 +5,8 @@ describe('Register', () => {
     // Input fields
     cy.get('input[name=firstname]').type('TestFirstname')
     cy.get('input[name=lastname]').type('TestLastname')
-    cy.get('input[name=mail]').type('mail35@test.com')
-    cy.get('input[name=phone]').type('35')
+    cy.get('input[name=mail]').type('mail44@test.com')
```

Comment:
Why did you change this ?

Reply:
In order to test the registration, I need to put a new e-mail and phone number.
In the future, I want to use https://github.com/marak/Faker.js/ to autogenerate the fields and have regression tests.

As I said in our conversation, that cypress is still in the early stages, you can ignore it for now.

Reply:
The way to go here, would be setting up a test environment (backend + database), use fixtures, and start each test with a clean database only containing the desired fixture

Reply:
We need to improve here. Let's create a separate story for that.